metrics/prometheus/prometheus.config.yml

```diff
@@ -6,4 +6,8 @@ scrape_configs:
       - targets: ["localhost:8080","localhost:3000"]
 
     metrics_path: /api/v1/metrics/
-    scheme: http
\ No newline at end of file
+    scheme: http
+    
+    authorization:
```

Comment:
can we update the docs as well? now that it requires an api key
https://docs.flowiseai.com/using-flowise/monitoring